jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -696,8 +688,34 @@
         # Increase qps limits in scheduler and controller manager.
         export CONTROLLER_MANAGER_TEST_ARGS="--kube-api-qps=100 --kube-api-burst=100"
         export SCHEDULER_TEST_ARGS="--kube-api-qps=100 --kube-api-burst=100"
-        # Increase throughput in Load test.
-        export LOAD_TEST_THROUGHPUT=50
-        export KUBE_NODE_OS_DISTRIBUTION="debian"
+
+    suffix:
+        - 'gce-enormous-cluster':  # kubernetes-e2e-gce-enormous-cluster
+            description: 'Starts and deletes empty 2000 node cluster and runs scalability tests on it. Does allow few Nodes to fail during startup.'
+            timeout: 420
+            job-env: |
+                export E2E_NAME="e2e-enormous-cluster"
+                export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Performance\] \
+                    --kube-api-content-type=application/vnd.kubernetes.protobuf \
+                    --allowed-not-ready-nodes=20"
+                # TODO: re-enable service creation after we debug problems with this suite
+                # export CREATE_SERVICES="true"
+                # Increase throughput in Load test.
+                export LOAD_TEST_THROUGHPUT=50
+                export KUBE_NODE_OS_DISTRIBUTION="debian"
+        - 'gce-enormous-deploy':  # kuberentes-e2e-gce-enormous-deploy
+            description: 'Starts up empty 2000 node cluster and waits until it is running. Used for manual testing.'
+            timeout: 180
+            job-env: |
+                export E2E_NAME="e2e-enormous-deploy"
+                export E2E_TEST="false"
```

Comment:
I started thinking about it and now I think that what we should do is to actually run this "Empty" test at the end to wait until all routes are created.

If so, we should probably also increase timeout.